src/entities/BlsBaseDSSLib.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity ^0.8.0;
+
+import "../interfaces/ICore.sol";
+import {BN254} from "./BN254.sol";
+import {ICore} from "../interfaces/ICore.sol";
+
+library BlsBaseDSSLib {
+    using BN254 for BN254.G1Point;
+
+    ///@notice the user of the library should initiate a storage variable that has been built on this struct
+    struct State {
+        ICore core;
+        mapping(address operatorAddress => bool exists) operatorExists;
+        mapping(address operatorAddress => uint256 index) operatorIndex;
+        address[] operatorAddresses;
```

Comment:
Use `BaseDSSLib.State` here instead and overload `baseDssStatePtr()` pointing to this.

Reply:
Need different state than BaseDSS, that's why made a new one